mage_integrations/mage_integrations/destinations/elasticsearch/__init__.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+import sys
+
+from elasticsearch import Elasticsearch as Elasticsearch_client
+
+from mage_integrations.destinations.base import Destination
+from mage_integrations.destinations.elasticsearch.target_elasticsearch.common import (
+    API_KEY,
+    API_KEY_ID,
+    BEARER_TOKEN,
+    HOST,
+    PASSWORD,
+    PORT,
+    SCHEME,
+    SSL_CA_FILE,
+    USERNAME,
+)
+from mage_integrations.destinations.elasticsearch.target_elasticsearch.target import (
+    TargetElasticsearch,
+)
+
+
+class Elasticsearch(Destination):
+    def _process(self, input_buffer) -> None:
+        self.config['state_path'] = self.state_file_path
+        TargetElasticsearch(config=self.config,
+                            logger=self.logger).listen_override(
+                            file_input=open(self.input_file_path, 'r')
+        )
+
+    def test_connection(self) -> None:
+        config = {}
+        scheme = self.config[SCHEME]
```

Comment:
Could you not duplicate the code? You can make `_authenticated_client` method public and call it.